tools/rna_tools/htseq-clip/.shed.yml

```diff
@@ -0,0 +1,14 @@
+name: htseq_clip
+owner: rnateam
+description: |
+  htseq-clip is a toolset for the analysis of eCLIP/iCLIP datasets
+homepage_url: https://github.com/EMBL-Hentze-group/htseq-clip
+long_description:
+  htseq-clip is a toolset for the analysis of eCLIP/iCLIP datasets. 
+  It can be used to generate files necessary for data analysis using 
+  the companion R/Bioconductor package DEWSeq.
+remote_repository_url: https://github.com/bgruening/galaxytools/tree/master/tools/rna_tools/htseq-clip
+type: unrestricted
+categories:
```

Comment:
@bgruening do you mean that you want to put `CLIP-seq` here already?

Reply:
@michauhl this is what I understood, correct?

Reply:
@bgruening @martenson if possible, yes.

Reply:
Sounds great, done.

Reply:
@michauhl check out https://toolshed.g2.bx.psu.edu/ and add your preferred categories :)

Reply:
@martenson @bgruening perfect, thank you! :)